src/shared/locales/en-US/window.js

```diff
@@ -2,7 +2,7 @@ export default {
   'reload': 'Reload',
   'close': 'Close',
   'minimize': 'Minimize',
-  'zoom': 'Zoom',
+  'zoom': 'Maximize',
```

Comment:
`Zoom` is the usual name on macOS. It is not only maximized. Please revert this change.

https://electronjs.org/docs/api/menu-item#roles